e2e/app/page/cohort-criteria-modal.ts

```diff
@@ -32,9 +35,10 @@ export enum FilterSign {
   GreaterThanOrEqualTo = 'Greater than or Equal to',
   LessThanOrEqualTo = 'Less than or Equal to',
   Between = 'Between',
+  GreaterThanEqualTo = 'Greater Than or Equal To',
```

Comment:
Just tagged you on a [pr](https://github.com/all-of-us/workbench/pull/3639) to fix the difference between components so we don't need multiple greater than options here. (PR also changes 'Less than or Equal to' -> 'Less Than or Equal To' )